### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by ttahm3d**
- Created grid

   - 2 Column and 3 column
   - Autofit grid

- Added codeblocks

### CODE REVIEW

File: css/components/sidebar.css [hunk 1]
```diff
@@ -21,22 +21,23 @@
   display: block;
   cursor: pointer;
   transition: all 0.2s linear;
-  border-radius: 0.25rem;
```
Comment (vaibhav-mahalle) [inline css/components/sidebar.css:24]: is there any reason to use rem and not px as it is less than 1 rem

**Review by vaibhav-mahalle — COMMENTED** [review]
looks good to me.

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go perfect code structure , CSS variable and class naming👍

File: css/components/sidebar.css (hunk 1 above)
Comment (ttahm3d) [reply css/components/sidebar.css:24]: I calculate everything down to rem even if it px except when it is 1 or 2px for border

**Review by vaibhav-mahalle — APPROVED** [review]

**Pull request merged**